common/math/doc.go

```diff
@@ -18,5 +18,11 @@
 // This file is derived from common/math/big.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package math provides integer math utilities.
+/*
+Package math provides convenience functions to use big.Int and to parse string into an integer.
+
+`big.go` provides Max, Min, Pow, Parse functions for big.Int types and it also implements encoding.TextMarshaler and encoding.TextUnmarshaler.
+
+`integer.go` provides functions to parse string into unsigned int and to calculate safely by detecting overflow. It also provides implementation of encoding.TextMarshaler and encoding.TextUnmarshaler.
```

Comment:
Line 24 and 26 both have "It also provides implementation of encoding.TextMarshaler and encoding.TextUnmarshaler.". It would be helpful if what encoding.TextMarshaler/TextUnMarshaler actually does.

Reply:
@ehnuje Thanks for your good feedback. Explanation of TextMarshaler/Unmashaler are added.